### DISCUSSION TITLE
Error when injecting

### DISCUSSION THREAD

**Discussion by KennySpark** [YimMenu/YimMenu#1356] [General]
Today my social club decided to update and now when i inject the menu i get an error "failed to find 'pd'" in the command prompt window. anybody know a fix? thanks

**Comment by KennySpark** [comment]
![yimenu error](https://github.com/YimMenu/YimMenu/assets/6549231/dcf52d50-1d9c-4de2-9038-e1302401b2b9)

**Reply by takoyakiwhite** [reply]
Try https://github.com/YimMenu/YimMenu/pull/1358

**Reply by KennySpark** [reply]
> Try #1358

Thanks. iv opened the pointers.cpp file in visual studio but when i change the code and build i get errors. any suggestions? sorry im new to vs

**Reply by takoyakiwhite** [reply]
> > Try #1358
> 
> Thanks. iv opened the pointers.cpp file in visual studio but when i change the code and build i get errors. any suggestions? sorry im new to vs

just using the built DLL
